### PR TITLE
feat uservice-dynconf: add codegen for SQL queries and JSONs

### PR DESCRIPTION
**Pull request by Dangerio**

### CODE REVIEW

File: src/handlers/admin_v1_configs.cpp [hunk 1]
```diff
@@ -23,15 +24,22 @@ struct RequestData {
...
-  result.kill_switches_enabled =
-      request["kill_switches_enabled"].As<std::unordered_set<std::string>>({});
-  result.kill_switches_disabled =
-      request["kill_switches_disabled"].As<std::unordered_set<std::string>>({});
+  if (body.service.has_value()) {
+    result.service = body.service.value();
+  }
+
   return result;
 }
```
Comment (Anton3) [inline src/handlers/admin_v1_configs.cpp:21]: `ConsitstsOfIdsFromConfigs -> ConsistsOfIdsFromConfigs`

File: src/handlers/admin_v1_configs.cpp [hunk 2]
```diff
@@ -23,15 +24,22 @@ struct RequestData {
 };
 
 RequestData ParseRequest(const userver::formats::json::Value &request) {
```
Comment (Anton3) [inline src/handlers/admin_v1_configs.cpp]: Let's delete `RequestData` and use the code-generated structs directly

File: src/handlers/admin_v1_configs.cpp [hunk 3]
```diff
@@ -1,12 +1,13 @@
 #include "admin_v1_configs.hpp"
+#include "docs/api/api.hpp"
```
Comment (Anton3) [inline src/handlers/admin_v1_configs.cpp]: Please use `<>` for all includes

File: src/handlers/admin_v1_configs_delete.cpp [hunk 4]
```diff
@@ -17,9 +19,11 @@ struct RequestData {
 };
 
 RequestData ParseRequest(const userver::formats::json::Value &request) {
```
Comment (Anton3) [inline src/handlers/admin_v1_configs_delete.cpp]: Let's delete `RequestData` and use the code-generated structs directly

**Review by Anton3 — CHANGES_REQUESTED** [review]

File: src/handlers/configs_values.cpp [hunk 5]
```diff
@@ -1,36 +1,17 @@
 #include "configs_values.hpp"
```
Comment (Anton3) [inline src/handlers/configs_values.cpp]: include groups (with a single empty line between each) should go as follows:
0. single pair include (if applicable)
1. std
2. userver
3. codegen
4. service

File: src/handlers/admin_v1_configs.cpp (hunk 1 above)
Comment (Dangerio) [reply src/handlers/admin_v1_configs.cpp:21]: Done

File: src/handlers/admin_v1_configs.cpp (hunk 2 above)
Comment (Dangerio) [reply src/handlers/admin_v1_configs.cpp]: Ok, done

File: src/handlers/admin_v1_configs.cpp (hunk 3 above)
Comment (Dangerio) [reply src/handlers/admin_v1_configs.cpp]: Ok, I have corrected includes in all files

File: src/handlers/admin_v1_configs_delete.cpp (hunk 4 above)
Comment (Dangerio) [reply src/handlers/admin_v1_configs_delete.cpp]: Ok, done

File: src/handlers/configs_values.cpp (hunk 5 above)
Comment (Dangerio) [reply src/handlers/configs_values.cpp]: Ok, I have fixed include order in all files

**Review by Anton3 — APPROVED** [review]

**Pull request merged**